src/Symfony/Component/Lock/Tests/Store/CombinedStoreTest.php

```diff
@@ -62,8 +62,8 @@ public function getStore()
     protected function setUp()
     {
         $this->strategy = $this->getMockBuilder(StrategyInterface::class)->getMock();
-        $this->store1 = $this->getMockBuilder([PersistingStoreInterface::class, BlockingStoreInterface::class])->getMock();
-        $this->store2 = $this->getMockBuilder([PersistingStoreInterface::class, BlockingStoreInterface::class])->getMock();
+        $this->store1 = $this->getMockBuilder(BlockingStoreInterface::class)->getMock();
+        $this->store2 = $this->getMockBuilder(BlockingStoreInterface::class)->getMock();
```

Comment:
Is there a reason to not use “createMock()” ?

Reply:
We should use `createMock()`. Existing tests might use this construct, but that's for historical reasons.